micro_ros_agent/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@ cmake_minimum_required(VERSION 3.5)
 option(UROSAGENT_GENERATE_PROFILE
   "Generates agent.refs according to the .msgs provided in the .repos" OFF
   )
+
+option(UGENERATE_GRAPH_MANAGER
+  "Include graph manager functionality" OFF
```

Comment:
IMO this should be defaulted to ON, but it will bring problems until the FastDDS version issue is solved. 
@jamoralp thoughts?

Reply:
Yes, that is right. It should be default to on, then update our colcon.meta files and the build_system scripts accordingly to switch it off until the Fast-DDS problem gets sorted